deps/build.jl

```diff
@@ -62,7 +62,8 @@ juliakspec = abspath(spec_name)
 binary_name = @windows? "julia.exe":"julia"
 kernelcmd_array = String[joinpath(JULIA_HOME,("$binary_name")), "-i"]
 ijulia_dir = get(ENV, "IJULIA_DIR", Pkg.dir("IJulia")) # support non-Pkg IJulia installs
-append!(kernelcmd_array, ["-F", joinpath(ijulia_dir,"src","kernel.jl"), "{connection_file}"])
+startupfile = VERSION >= v"0.4-" ? "--startup-file=yes" : "-F"
```

Comment:
`0.4`, not `0.4-` (which denotes every prerelease of 0.4, from the instant it was forked from 0.3)